config/data/sites/sites.yaml

```diff
@@ -43,7 +43,7 @@ sites:
 site:
     US:
         general:
-          url: "${general.baseUrl}/"
+          url: "${general.baseUrl}"  
```

Comment:
Same thing. Can undo this change as well before pushing. We want the suite as it was before. Except major test related changes.